src/sota_tools/ostree_object.cc

```diff
@@ -241,7 +247,24 @@ void OSTreeObject::Upload(const TreehubServer &push_target, CURLM *curl_multi_ha
   request_start_time_ = std::chrono::steady_clock::now();
 }
 
-void OSTreeObject::PresenceUnknown(RequestPool &pool, const int64_t rescode) {
+void OSTreeObject::CheckChildren(RequestPool &pool, const long rescode) {  // NOLINT
```

Comment:
This clang-tidy lint is a bit annoying https://clang.llvm.org/extra/clang-tidy/checks/google-runtime-int.html. Even if I agree with the intent, it is a bit strict.

Maybe we could disable it. Otherwise, maybe use the specific lint id: `// NOLINT(google-runtime-int)`

Reply:
Generally, I like it, it's just because of the curl API that we have the problem. You are right about specifying the lint ID, though.